bubblewrap.c

```diff
@@ -1730,6 +1732,23 @@ parse_args_recurse (int          *argcp,
 
           opt_sync_fd = the_fd;
 
+          argv += 1;
+          argc -= 1;
+        }
+      else if (strcmp (arg, "--close-fd") == 0)
+        {
+          int the_fd;
+          char *endptr;
+
+          if (argc < 2)
+            die ("--close-fd takes an argument");
+
+          the_fd = strtol (argv[1], &endptr, 10);
+          if (argv[1][0] == 0 || endptr[0] != 0 || the_fd < 0)
+            die ("Invalid fd: %s", argv[1]);
+
+          opt_close_fd = the_fd;
```

Comment:
This supports just one fd?  I think to be a bit more general we should support multiple right?

Reply:
I mean we could, but why would you ever want that? This is like a single place that triggers when the entire sandbox is done, and you only need to know that once.

Reply:
Ah.  Isn't this then `--notify-setup-complete-fd` or something?

Reply:
@cgwalters yeah, probably